modules/qna/src/views/full/index.tsx

```diff
@@ -44,9 +44,22 @@ export default class QnaAdmin extends Component<Props> {
 
   fetchFlows() {
     this.props.bp.axios.get('/flows').then(({ data }) => {
-      const flowsList = data
-        .filter(flow => !flow.name.startsWith('skills/'))
-        .map(({ name }) => ({ label: name, value: name }))
+      const reorderFlows = flows =>
+        [
+          flows.find(x => x.name === 'main'),
+          flows.find(x => x.name === 'error'),
+          flows.find(x => x.name === 'timeout'),
+          ...flows.filter(x => !['main', 'error', 'timeout'].includes(x.name))
+        ].filter(x => Boolean(x))
+
+      const getFlowLabel = (name: string) =>
+        name === 'main' ? 'Main' : name == 'error' ? 'Error handling' : name == 'timeout' ? 'Timeout' : name
```

Comment:
Please use an if/else statement when there are multiple possibilities, a bunch of ? : is a bit hard to read